tools/rapids-rattler-channel-string

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+RAPIDS_CHANNEL="rapidsai-nightly"
```

Comment:
```suggestion
RAPIDS_CHANNEL="${RAPIDS_CHANNEL:-rapidsai-nightly}"
```
Minor thing, but might make it easier to toy with alternate channels.

Reply:
I’m guessing that anywhere we would use this, we would have (and would want) the ability to override the whole channel set. I’m not sure if this flexibility buys us very much - but happy to hear others’ thoughts as well.

Reply:
I do mean other env vars, too, but I agree that I'm not sure that the flexibility buys us much. Still, the NVAIE LWS builds have made me extremely grateful for this kind of flexibility in various places. Even though the normal open-source-facing builds would never use this stuff, it may be useful in other scenarios.

Reply:
In this _particular_ context, I think the escape valve is setting `$RATTLER_CHANNELS` explicitly, but I'm happy to revisit if we find ourselves needing to tweak it!

Reply:
I agree, I think we'll always be in a position to just not use this tool and/or replace the channels manually.